### PR TITLE
Update React Native Expo shell commands

### PR DESCRIPTION
**Pull request by krollins-mdb**
## Pull Request Info

- [Bootstrap with Expo](https://preview-mongodbkrollinsmdb.gatsbyjs.io/realm/expo-update/sdk/react-native/bootstrap-with-expo/): Updated shell commands for creating and running the Expo app.

### Reminder Checklist

Before merging your PR, make sure to check a few things.

- [x] Did you tag pages appropriately?
  - genre
  - meta.keywords
  - meta.description
- [x] Describe your PR's changes in the Release Notes section
- [x] Create a Jira ticket for related docs-app-services work, if any

### Release Notes

- **React Native SDK**
  - Bootstrap with Expo: Update shell commands and clarify how to get an Expo app working.

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/sdk/react-native/bootstrap-with-expo.txt [hunk 1]
```diff
@@ -4,24 +4,28 @@
...
    :backlinks: none
    :depth: 2
    :class: singlecol
 
-The :npm:`Realm Expo template <package/@realm/expo-template>`
+The :npm:`Atlas Device SDK Expo template <package/@realm/expo-template>`
 provides a fully working React Native application that you can use to bootstrap
-your app development project with Realm. This documentation covers how to
-initialize and work with the Realm Expo template.
+your app development project with Atlas Device SDK. This documentation covers
```
Comment (cbullinger) [inline source/sdk/react-native/bootstrap-with-expo.txt:19]:
```suggestion
your app development project with Atlas Device SDK. This guide covers
```
'guide' seems more appropriate here (and less formal)

File: source/sdk/react-native/bootstrap-with-expo.txt [hunk 2]
```diff
@@ -4,24 +4,28 @@
...
--  `Expo <https://docs.expo.dev/>`_, a framework to develop, build and deploy
-   React Native applications quickly.
-- :npm:`@realm/react <package/@realm/react>`, an NPM package that provides an
-   easy-to-use API to perform common Realm operations, such as querying or
-   writing to a realm and listening to realm objects.
+- `Expo <https://docs.expo.dev/>`_: a framework for developing, building and
+  deploying React Native applications.
+- :npm:`@realm/react <package/@realm/react>`: an npm package that provides an
+  easy-to-use API to perform common Realm operations, such as querying or
+  writing to a realm and listening to realm objects.
```
Comment (cbullinger) [inline source/sdk/react-native/bootstrap-with-expo.txt:28]:
```suggestion
  writing to a database and listening to database objects.
```
suggestion

File: source/sdk/react-native/bootstrap-with-expo.txt [hunk 3]
```diff
@@ -30,32 +34,27 @@ Prerequisites
...
-   npm install -g expo-cli
-
-
 Initialize the Template
 -----------------------
-To initialize a React Native application using the Realm Expo template use the Expo CLI.
 
-Use ``expo init``  with the flag ``--template @realm/expo-template`` to initialize a
-React Native application using the Realm Expo Template. 
+To initialize the template application use ``create-expo-app``. Pass the flag
```
Comment (cbullinger) [inline source/sdk/react-native/bootstrap-with-expo.txt:43]:
```suggestion
To initialize the template application, use ``create-expo-app``. Pass the flag
```

File: source/sdk/react-native/bootstrap-with-expo.txt [hunk 4]
```diff
@@ -30,32 +34,27 @@ Prerequisites
...
-   expo init MyAwesomeRealmApp --template @realm/expo-template
+   npx create-expo-app AtlasDeviceSdkApp --template @realm/expo-template
+
+Install app dependencies:
+
+.. code-block:: shell
+
+   npm install
 
 Explore The File Structure
```
Comment (cbullinger) [inline source/sdk/react-native/bootstrap-with-expo.txt:59]:
```suggestion
Explore the File Structure
```

File: source/sdk/react-native/bootstrap-with-expo.txt [hunk 5]
```diff
@@ -81,7 +80,8 @@ The relevant files are as follows:
        the realm.
 
    * - TaskList.tsx
-     - A React component that gets a list of Task objects from the realm and renders each object as a ``TaskItem``  component.
+     - A React component that gets a list of Task objects from the realm and
```
Comment (cbullinger) [inline source/sdk/react-native/bootstrap-with-expo.txt:83]:
```suggestion
     - A React component that gets a list of Task objects from the database and
```
suggestion

File: source/sdk/react-native/bootstrap-with-expo.txt [hunk 6]
```diff
@@ -112,7 +112,7 @@ without waiting for the native build process again, you can simply run:
 
 .. code-block:: shell
    
-   expo start --dev-client
+   npx expo start --dev-client
 
 However, if you made any `changes to the native code
 <https://docs.expo.dev/workflow/customizing/>`_, you will have to re-run ``expo
```
Comment (cbullinger) [inline source/sdk/react-native/bootstrap-with-expo.txt:118]: should these also be `npx expo run:ios` and `npx expo run:android` (on this and next line)?

**Review by cbullinger — APPROVED** [review]

File: source/sdk/react-native/bootstrap-with-expo.txt [hunk 7]
```diff
@@ -4,24 +4,28 @@
 Bootstrap with Expo - React Native SDK
 ======================================
 
+.. meta::
+  :description: Create a React Native Expo app that uses the Atlas Device SDK.
+  :keywords: Expo
```
[the remark this answers is not in the document]
Comment (krollins-mdb) [reply source/sdk/react-native/bootstrap-with-expo.txt:9]: Yeah, you're right. We don't really need it.

File: source/sdk/react-native/bootstrap-with-expo.txt (hunk 6 above)
Comment (krollins-mdb) [reply source/sdk/react-native/bootstrap-with-expo.txt:118]: Yep! Thank you for noticing that I missed this.

**Pull request merged**